package/Dockerfile.subctl

```
@@ -1,8 +1,21 @@
-FROM scratch
+ARG BASE_BRANCH
+ARG SOURCE=/go/src/github.com/submariner-io/submariner-operator
+
+FROM --platform=${BUILDPLATFORM} quay.io/submariner/shipyard-dapper-base:${BASE_BRANCH} AS builder
+ARG SOURCE
+ARG TARGETPLATFORM
+
+COPY . ${SOURCE}
+
+RUN rm -f cmd/bin/subctl && make -C ${SOURCE} LOCAL_BUILD=1 cmd/bin/${TARGETPLATFORM}/subctl
```

```suggestion
RUN rm -f cmd/bin/subctl && make -C ${SOURCE} cmd/bin/${TARGETPLATFORM}/subctl
```

There's no significance to `LOCAL_BUILD` here since it's already inside the shipyard container env.